CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@ Add: httpsPrepare.sh script to contextBroker-tests RPM (Issue #2)
 Fix:  Fixed a bug about generic error handling for API version 2 (Issue #1087)
 Fix:  Fixed a bug about error handling of invalid Service-Path for API version 2 (Issue #1092)
 Add:  Implemented a RAM-cache for patterned ngsi10 subscriptions (Issue #1048)
+Add:  POST /v2/entities/<entityId> - append or update attributes (by entity ID) under version 2 (Issue #981)
```

Comment:
In addition, I understand that the PR also implements #983 (i.e. ?op=append). In that case, it also should be mentioned in the CNR (maybe in a separate line).